essentials-framework/Essentials Core/PepperDashEssentialsBase/Fusion/EssentialsHuddleSpaceFusionSystemControllerBase.cs

```diff
@@ -1676,20 +1676,46 @@ public static StringSigData CreateOffsetStringSig(this FusionRoom fr, uint numbe
         public static FusionStaticAsset CreateStaticAsset(this FusionRoom fr, uint number, string name, string type,
             string instanceId)
         {
-            Debug.Console(0, "Adding Fusion Static Asset '{0}' to slot {1} with GUID: '{2}'", name, number, instanceId);
+            try
+            {
+                Debug.Console(0, "Adding Fusion Static Asset '{0}' to slot {1} with GUID: '{2}'", name, number, instanceId);
 
-            fr.AddAsset(eAssetType.StaticAsset, number, name, type, instanceId);
-            return fr.UserConfigurableAssetDetails[number].Asset as FusionStaticAsset;
+                fr.AddAsset(eAssetType.StaticAsset, number, name, type, instanceId);
+                return fr.UserConfigurableAssetDetails[number].Asset as FusionStaticAsset;
+            }
+            catch (InvalidOperationException ex)
+            {
+                Debug.Console(0, Debug.ErrorLogLevel.Notice, "Error creating Static Asset for device: '{0}'.  Check that multiple devices don't have missing or duplicate uid properties in configuration. /r/nError: {1}", name, ex);
+                return null;
+            }
+            catch (Exception e)
+            {
+                Debug.Console(2, "Error creating Static Asset: {0}", e);

```

Comment:
We should probably dump this to the Error log as well, in case there's an exception and debug isn't at 2...